recipes/hyperspyui/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "hyperspyUI" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c666d44e6330cd535590d00e7a11163d200b286ac99f8686aef8d94d063bf635
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  entry_points:
+    - hyperspyui = hyperspyui.__main__:main
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+
+  run:
+    - python >=3.5
+    - hyperspy-gui-traitsui
+    - pyqode.python
+    - autopep8
+    - pep8
+    - qtconsole
+    - qtpy
+    - ffmpeg
+
+test:
+  imports:
+    - hyperspyui
+    - hyperspyui.__main__
+
+  # commands:
+  #  - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'hyperspyui -h'
```

Comment:
I disabled these tests for now.

Reply:
It may be worth explaining why the tests are disabled in the comments.

Reply:
It might be possible to fix them by creating a file in the recipe directory [like this](https://github.com/pymor/staged-recipes/blob/6faef0388922acc898615a0c130c4cabdd809940/recipes/pymor/yum_requirements.txt). (You might only need xvfb and not the other two items.)

Reply:
Thanks @chrisburr , but isn't  ``xorg-x11-server-Xvfb`` already in ``yum_requirements.txt``?

Reply:
Indeed it is! 😆 In that case I'm also not sure why these tests are disabled.

Reply:
I copied paste this from another recipe just to give a try but I haven't look into details on why it doesn't work. Since all the CI services running in this PR and we only needs circle ci, I would prefer to work it out in the feedstock repo rather than here.

Reply:
Then LGTM 👍